tests/fixtures.py

```diff
@@ -17,11 +20,25 @@ def __init__(
         important_parameter: Optional[str] = None,
         data_path: Optional[str] = None,
         data_mapping: Dict[str, str] = {},
+        risk_bound: float = None,
```

Comment:
It might be nice to pack these into an `optimize_kwargs` dictionary. This will make it easier to know which additional arguments are necessary for which interface test methods.

To make this consistent, if you could also pack `calibrate_kwargs`, etc. into a similar dictionary that would be great.

Reply:
I have added the optimize_kwargs. Can you check this to see if this is what you meant